src/client/pages/AssessmentSection/DataTable/Table/Row/RowData/Cell/hooks/useOnChange.ts

```diff
@@ -85,37 +85,45 @@ export default (props: Props): UseOnChange => {
     const { clipboardData } = event
     const el = document.createElement('html')
     el.innerHTML = clipboardData.getData('text/html')
+    const rowsToPaste = el.getElementsByTagName('tr')
 
-    const rows = el.getElementsByTagName('tr')
-    const rowSpecs = table.rows.filter((row) => ![RowType.header, RowType.noticeMessage].includes(row.props.type))
+    if (rowsToPaste.length > 0) {
```

Comment:
if length is 0 can u return to avoid wrap?

Reply:
```
 else {
      const value = clipboardData.getData('text/plain')
      _persistSanitizedValue({
        ...nodeValue,
        raw: value,
      })
    }
```